src/recommender/actions/train.py

```diff
@@ -15,6 +15,10 @@ def apply(self, ir: IR) -> IR:
             return
         comment = Comment()
 
+        num_nodes = int(ir.compute_config.get("num_nodes", 1))
```

Comment:
lets define the defaults `1` and use the variable here. You can create a constants.py file